.github/workflows/amazon-aarch64.yml

```diff
@@ -59,3 +59,10 @@ jobs:
         with:
           name: ${{ env.NOW }}-amazon-aarch64-${{ github.run_id }}
           path: archive
+
+      - name: Send notification
+        if: failure()
+        uses: ./.github/actions/failure_notify
```

Comment:
Usually, we use dash instead of underscore.

Reply:
fixed